### PR TITLE
Preparing spec doc v3.1

### PR DESCRIPTION
**Pull request by mkarg**
Collection of useful steps towards spec doc v3.1

**As these are no API changes, this is a fast-track review period of just one day as per our [committer rules](https://github.com/eclipse-ee4j/jaxrs-api/wiki/Committer-Conventions#minimum-length-of-review-period-before-merge--close-of-prs-and-closing-of-issues).**

### CODE REVIEW

File: jaxrs-spec/src/main/asciidoc/chapters/introduction/_project_team.adoc [hunk 1]
```diff
@@ -17,8 +17,6 @@ of the project team and various contributors.
 
 The following are the current committers of the Jakarta RESTful Web Services specification project:
 
--- Sebastian Daschner (IBM)
```
Comment (chkal) [inline jaxrs-spec/src/main/asciidoc/chapters/introduction/_project_team.adoc:20]: I'm wondering why you removed Sebastian as he is still listed here: https://projects.eclipse.org/projects/ee4j.jaxrs/who

**Comment by andymc12** [comment]
It looks like the CI tasks are not running for this PR and at least a few of the others too.  For these PRs that upgrade build tools, I'd really like to see the CI pass to ensure that some tool upgrade doesn't have a regression that breaks us.

**Comment by mkarg** [comment]
> It looks like the CI tasks are not running for this PR and at least a few of the others too. For these PRs that upgrade build tools, I'd really like to see the CI pass to ensure that some tool upgrade doesn't have a regression that breaks us.

Me too. Already asked the EF admins for help.

File: jaxrs-spec/src/main/asciidoc/chapters/introduction/_project_team.adoc (hunk 1 above)
Comment (mkarg) [reply jaxrs-spec/src/main/asciidoc/chapters/introduction/_project_team.adoc:20]: Sebastian was inactive for very long time and has not contributed to 3.1. That listing on the EF page seems to be rather politically motivated, so the EF never throws out a committer there. I do not see the value of still having him mentioned in the *current* spec document. If I missed some of his merits on the 3.1 time frame, please point me to his actual contributions (issues, review, etc.). I would be happy to keep his name in the spec if he *actually* contributed to 3.1.

**Comment by mkarg** [comment]
> > It looks like the CI tasks are not running for this PR and at least a few of the others too. For these PRs that upgrade build tools, I'd really like to see the CI pass to ensure that some tool upgrade doesn't have a regression that breaks us.
> Me too. Already asked the EF admins for help.

See https://bugs.eclipse.org/bugs/show_bug.cgi?id=574335. It seems the EF needs to migrate the repositories from travis.org to travis.com. I did that for my persoanl JAX-RS fork and force-pushed to this PR (BTW, all committers should do that if they want to use travis), so @andymc12 here you have an *interim* proof that this PR passes fine: https://travis-ci.com/github/mkarg/jaxrs-api/builds/230976115.

File: jaxrs-spec/src/main/asciidoc/chapters/introduction/_project_team.adoc (hunk 1 above)
Comment (spericas) [reply jaxrs-spec/src/main/asciidoc/chapters/introduction/_project_team.adoc:20]: @mkarg The decision to remove someone should be discussed. Once again, you are introducing changes in a PR that are _not related_ to the PR's title or intent. I'm fine with discussing this but in a separate PR.

**Review by spericas — CHANGES_REQUESTED** [review]
Do not remove contributors as part of this type of PR.

File: jaxrs-spec/src/main/asciidoc/chapters/introduction/_project_team.adoc (hunk 1 above)
Comment (mkarg) [reply jaxrs-spec/src/main/asciidoc/chapters/introduction/_project_team.adoc:20]: Then intention of this PR *is* to discuss it. What is the benefit of discussing this single commit in *different* PR? The arguments will be the same.

Comment (spericas) [reply jaxrs-spec/src/main/asciidoc/chapters/introduction/_project_team.adoc:20]: It isn't based on the title or the intent of this PR. If the scope of an update is different from the title/intent, it should be a different PR/discussion. Please create another PR that isn't related to the _technical_ intent of the PR.

Comment (mkarg) [reply jaxrs-spec/src/main/asciidoc/chapters/introduction/_project_team.adoc:20]: While I do not see your argument actually (the text in the subject of this PR does not say this is just about "technical" changes; moreover it actually says this is just a bunch of changes) we can separate the PRs if you have such strong feelings about it.

**Comment by mkarg** [comment]
@spericas Done with requested changes, please withdraw your veto.

@chkal @andymc12 PR was updated, so please cast your vote again.

File: jaxrs-spec/src/main/asciidoc/chapters/introduction/_project_team.adoc (hunk 1 above)
Comment (mkarg) [reply jaxrs-spec/src/main/asciidoc/chapters/introduction/_project_team.adoc]: Undone committer removal, opened separate PR for this. Please withdraw veto. Thanks.

**Review by chkal — APPROVED** [review]

**Review by andymc12 — APPROVED** [review]
LGTM

**Comment by mkarg** [comment]
@spericas Can I merge this or do you want more changes?

**Comment by mkarg** [comment]
@spericas You did not respond for another week and all your requested changes are done since ten days, so I am confused what's up. **Please respond ASAP.**

**Comment by spericas** [comment]
> @spericas You did not respond for another week and all your requested changes are done since ten days, so I am confused what's up. **Please respond ASAP.**

Sorry, was out on vacation. Will update now.

**Review by spericas — APPROVED** [review]

**Pull request merged**